specification/security/resource-manager/Microsoft.Security/preview/2022-01-01-preview/ServerVulnerabilityAssessmentsSettings.json

```diff
@@ -0,0 +1,328 @@
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/serverVulnerabilityAssessmentsSettings/{settingKind}": {
+      "get": {
+        "x-ms-examples": {
+          "Get the server vulnerability assessments setting of the kind settingKind that is set on the subscription": {
+            "$ref": "./examples/ServerVulnerabilityAssessmentsSettings/GetServerVulnerabilityAssessmentsSetting_example.json"
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Get a server vulnerability assessments setting of the requested kind, that is set on the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SettingKind"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerVulnerabilityAssessmentsSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Set a server vulnerability assessments setting of the kind settingKind on the subscription": {
+            "$ref": "./examples/ServerVulnerabilityAssessmentsSettings/PutServerVulnerabilityAssessmentsSetting_example.json"
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Create a server vulnerability assessments setting of the requested kind on the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SettingKind"
+          },
+          {
+            "$ref": "#/parameters/SettingBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerVulnerabilityAssessmentsSetting"
+            }
+          },
```

Comment:
PUT must be able to return 201 also, unless you're not allowed to create them because they're always there.